aodntools/timeseries_products/velocity_aggregated_timeseries_template.json

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
I imagine you probably have a lot of templates at the moment - maybe a time for refactoring?

For example, a lot of fields are probably repeats from other templates - maybe a template to rule them all, while small templates only put stuff to "overwrite" bits. This may also avoid spread typos, slightly different entries,  and help in maintenance (imagine something need to change - you have to remember and change all templates...).

Reply:
it is in the plan